prettier.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  singleQuote: true
```

Comment:
Are we gonna continue to hold out on this singleQuote preference?  I remember you wanting to use double-quotes, and I heard Austin want to stick with default prettier settings as well. :]

Reply:
For consistency cross all our existing projects I think yeah. I've gotten used to it (mostly) at this point. This is also in our internal prettier configs too